CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.4.1]
```

Comment:
@jhkennedy Can we add a note that the -gunw workflow we are still verifying the results?

Reply:
@dbekaert good idea! Will do!